sentry_sdk/_types.py

```diff
@@ -1,7 +1,4 @@
-try:
-    from typing import TYPE_CHECKING
-except ImportError:
-    TYPE_CHECKING = False
```

Comment:
I removed the try-except block since we are now supporting Python 3.6 and above.